### PR TITLE
Merge our gitignore with the RxJava his gitignore

### PR DESCRIPTION
**Pull request by Lamartio**
When developing I found that `.DS_Store` and `out/` were able to commit. Now I merged the gitignore of RxJava, but an alternative could be to use [gitignore.io](https://gitignore.io/).

### CODE REVIEW

**Comment by CherryPerry** [comment]
Hi, don't you think that such `.gitignore` is an overkill? We don't have any packages produced by out builds, compiled source is in already ignored `build` and `out` folder. The whole `.idea` folder with project files is ignored too. We don't use PMD, it is a Kotlin project. Neither NetBeans nor Eclipse. Maybe just add `.DS_Store` and `out/`?

**Comment by Lamartio** [comment]
Could be ;) Usually I pull in a template, but a more incremental way is also fine

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@
 .idea
 /build
 /classes
+out/
```
Comment (arkivanov) [inline .gitignore]: I'm not sure but shouldn't it be "/out"?

Comment (Lamartio) [reply .gitignore]: I picked this one from a template and never had troubles with it before. As for this project, I think both choices are fine.

Comment (CherryPerry) [reply .gitignore]: @arkivanov is right. Tried `out/` instead of `/out' and `com.badoo.reaktive.out.Test` is ignored.

**Pull request merged**